server.js

```diff
@@ -63,6 +65,18 @@ app.use(function(req, res, next) {
     next();
 });
 
+mailer.extend(app, {
+  from: 'Venderapp',
+  host: 'smtp.gmail.com', // hostname
+  secureConnection: true, // use SSL
+  port: 465, // port for secure SMTP
+  transportMethod: 'SMTP', // default is SMTP. Accepts anything that nodemailer accepts
+  auth: {
+  user: 'radienerga@gmail.com', // gmail id
```

Comment:
Are we using radienerga@gmail.com to send the Verification Mails?
After integrating with AWS servers, will we still use the express-mailer?

Reply:
As of now yes,....
I am not sue if we have a separate mailing server in AWS.